ios/RNTPTDocumentViewModule.m

```diff
@@ -85,7 +85,7 @@ - (NSError *)errorFromException:(NSException *)exception
                  rejecter:(RCTPromiseRejectBlock)reject)
 {
     @try {
-        NSString *path = [[self documentViewManager] exportAsImageForDocumentViewTag:tag pageNumber:pageNumber dpi:dpi imageFormat:(NSString*)imageFormat];
+        NSString *path = [[self documentViewManager] exportAsImageForDocumentViewTag:tag pageNumber:pageNumber dpi:dpi imageFormat:imageFormat];
```

Comment:
Everywhere else the param `imageFormat` in this method is referred to as `exportFormat`. Should this one be `exportFormat` too, either for correctness or consistency?